Lib/test/support/__init__.py

```diff
@@ -532,12 +532,26 @@ def requires_debug_ranges(reason='requires co_positions / debug_ranges'):
 is_emscripten = sys.platform == "emscripten"
 is_wasi = sys.platform == "wasi"
 
-has_fork_support = hasattr(os, "fork") and not is_emscripten and not is_wasi
+# Apple mobile platforms (iOS/tvOS/watchOS) are POSIX-like but do not
+# have subprocess or fork support.
+is_apple_mobile = sys.platform in {"ios", "tvos", "watchos"}
+is_apple = is_apple_mobile or sys.platform == "darwin"
+
+has_fork_support = (
+    hasattr(os, "fork")
+    and not is_emscripten
+    and not is_wasi
+    and not is_apple_mobile
```

Comment:
```suggestion
    and not (is_emscripten or is_wasi or is_apple_mobile)
```

Reply:
I'll defer to the opinion of the core team, but for my money, the "one per line" format is more readable.

Reply:
Agreed on the one item per line – my suggestion was more about switching from `and` to `or`.

Reply:
Makes sense - I've made that simplification.